app/src/components/LayerControlBody.vue

```diff
@@ -296,7 +353,12 @@ export default {
       scaleMax: 10,
       scaleStep: 0.01,
 
-      testValue: 0
+      testValue: 0,
+
+      selectedResolutionX: 0,
+      selectedResolutionY: 0,
+      selectedResolutionZ: 0,
```

Comment:
I think these should be computed properties, rather than data. 

They can be computed based on scale and selectedIncomingVolumeResolution{X,Y,Z}